parsons/action_kit/action_kit.py

```diff
@@ -536,6 +541,16 @@ def bulk_upload_table(self, table, import_page, autocreate_user_fields=0):
                 res: requests http response object
         """
         import_page = check_env.check('ACTION_KIT_IMPORTPAGE', import_page)
-        return self.bulk_upload_csv(table.to_csv(temp_file_compression='gzip'),
+        compress_maybe = dict()
+        user_fields_only = int(not any([
+            h for h in table.table.header()
```

Comment:
Could you add a line in the docstring above indicating that tables with only the user fields will be fast processed? Just to call people's attention to it.